utils/world_languages.py

```diff
@@ -98,14 +98,14 @@ def save_languages_to_json(languages: Set, output_file_name: str = file_name):
           None.
     """
 
-    if not path.isdir(file_path):
-        mkdir(file_path)
+    if not path.isdir(WORLD_LANG_PATH):
+        mkdir(WORLD_LANG_PATH)
     json_str = json.dumps(list(languages))
-    with open(file_path_and_name, "w") as f:
+    with open(WORLD_LANG_PATH_AND_NAME, "w") as f:
         f.write(json_str)
 
 
-def get_languages(filename: str = file_name) -> Set:
+def get_languages(filename: str = WORLD_LANG_FILENAME) -> Set:
```

Comment:
you shouldn't use the global variable. you should use the method's local variable, which is `output_file_name`. it is kind of a nasty bug :)

also if you find a bug, it is always a good idea to write a unit test that covers it. it reduces the changes of regression.

Reply:
I did as a mistake of course.